python_modules/libraries/dagster-tableau/dagster_tableau_tests/conftest.py

```diff
@@ -141,9 +141,9 @@ def sign_in_fixture():
 
 
 @pytest.fixture(name="get_workbooks", autouse=True)
-def get_workbooks_fixture():
+def get_workbooks_fixture(build_workbook_item):
     with patch("dagster_tableau.resources.BaseTableauClient.get_workbooks") as mocked_function:
-        mocked_function.return_value = SAMPLE_WORKBOOKS
+        mocked_function.return_value = [build_workbook_item()]
```

Comment:
will both the single and many workbook case be tested?

Reply:
I updated the description - the problem was that parsing the XML response received using the tableauserverclient  to a JSON object was different for one or many workbooks. This was not a problem when using the REST api because we were already recieving JSON responses.

Our implementation already supported many workbooks, like here because we pass a list of workbooks.

Reply:
I can plan adding more tests though.